tests/foreman/api/test_hostcollection.py

```diff
@@ -37,6 +48,35 @@ def setUpClass(cls):
         super(HostCollectionTestCase, cls).setUpClass()
         cls.org = entities.Organization().create()
         cls.hosts = [entities.Host(organization=cls.org).create() for _ in range(2)]
+        cls.lce = entities.LifecycleEnvironment(organization=cls.org).create()
+        cls.content_view = entities.ContentView(organization=cls.org).create()
+        cls.content_view.publish()
+        cls.content_view = cls.content_view.read()
+        promote(cls.content_view.version[0], environment_id=cls.lce.id)
+        cls.activation_key = entities.ActivationKey(
+            content_view=cls.content_view, environment=cls.lce, organization=cls.org
+        ).create()
+        setup_org_for_a_rh_repo(
+            {
+                'product': PRDS['rhel'],
+                'repository-set': REPOSET['rhst7'],
+                'repository': REPOS['rhst7']['name'],
+                'organization-id': cls.org.id,
+                'content-view-id': cls.content_view.id,
+                'lifecycle-environment-id': cls.lce.id,
+                'activationkey-id': cls.activation_key.id,
+            },
+            force_manifest_upload=True,
+        )
+        cls.custom_entities = setup_org_for_a_custom_repo(
```

Comment:
As far as I can tell, new test is not using this information. Why it's here, and is it really needed?

Reply:
It was there from before I reworked the test to use VMs, should have deleted it before, sorry, I was battling with other parts.